pkgs/applications/blockchains/ton/default.nix

```diff
@@ -35,8 +37,10 @@ stdenv.mkDerivation rec {
   buildInputs = [
     gperf
     libmicrohttpd
+    libsodium
     openssl
     readline
+    secp256k1
     zlib
   ];
 
```

Comment:
Seems broken on darwin
```
error: aligned allocation function of type 'void *(std::size_t, std::align_val_t)' is only available on macOS 10.13 or newer
```
Adding `broken = stdenv.isDarwin;` should be good enough for now

Reply:
Added `broken = ` mark on darwin.